Projekty/Projekt2/VinichenkoYevheniiWolnyKrzysztof/eda.ipynb

```diff
@@ -0,0 +1,2913 @@
+{
```

Comment:
<p>Nie jest to zbyt czytelne, ale rozumiem idee. Może wyłącznie xticks, yticks i grid by trochę poprawiło czytelność.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/322/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Teraz zauważyłem, że tam nie ma zupełnie podpisów pod wykresami przez co nawet jeżeli widzimy jakiś rozkład to nie wiadomo do jakiej zmiennej się on odnosi.